template/.github/workflows/build.yml.j2

```diff
@@ -379,3 +379,70 @@ jobs:
         run: ./preflight-linux-amd64 check container "$IMAGE_TAG" > preflight.out
       - name: "Passed?"
         run: '[ "$(./preflight-linux-amd64 check container "$IMAGE_TAG" | jq -r .passed)" == true ]'
+
+  openshift_tests:
+    name: Run integration tests
+    if: ${{ !github.event.pull_request.head.repo.fork }}
+    needs:
+      - package_and_publish
+    strategy:
+      fail-fast: false
+      matrix:
+        cluster: [
+          {
+            distribution: openshift,
+            version: 4.14.0-okd,
+            nodes: 3,
+            instances: r1.2xlarge
+          }]
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c # tag=v5.0.0
+        with:
+          python-version: '3.11'
+      - run: pip install beku-stackabletech
+
+      - name: Install test requirements
+        run: |
+          # Install requirements for the run_tests.sh script.
+          PATH=/usr/bin:$PATH
+
+          wget --no-verbose https://github.com/stackabletech/stackable-cockpit/releases/download/stackablectl-23.11.3/stackablectl-x86_64-unknown-linux-gnu
```

Comment:
Is it worth including the stackablectl version in the `env` section at the top, as it may change semi-regularly?